statsmodels/base/_penalized.py

```diff
@@ -0,0 +1,194 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun May 10 08:23:48 2015
+
+Author: Josef Perktold
+License: BSD-3
+"""
+
+import numpy as np
+from ._penalties import NonePenalty
+from statsmodels.tools.numdiff import approx_fprime_cs, approx_fprime
+
+class PenalizedMixin(object):
```

Comment:
One more line of whitespace before class definition please, if you happen to be making edits to this file anyway.